src/NServiceBus.AzureFunctions.StorageQueues/NServiceBus.AzureFunctions.StorageQueues.csproj

```
@@ -7,8 +7,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="NServiceBus.Azure.Transports.WindowsAzureStorageQueues" Version="8.1.4" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Storage" Version="3.0.6" />
+    <PackageReference Include="NServiceBus.Azure.Transports.WindowsAzureStorageQueues" Version="8.2.2" />
```

I guess we should switch to version ranges here
```suggestion
    <PackageReference Include="NServiceBus.Azure.Transports.WindowsAzureStorageQueues" Version="[8.2.2, 9)" />
```